src/test/java/net/minecraftforge/debug/misc/CustomTagTypesTest.java

```diff
@@ -234,4 +253,24 @@ public String getName()
             return "Mob Effect Tags";
         }
     }
+
+    public static class StructureFeatureTags extends ForgeRegistryTagsProvider<StructureFeature<?>> {
+
+        public StructureFeatureTags(DataGenerator gen, @Nullable ExistingFileHelper existingFileHelper)
+        {
+            super(gen, ForgeRegistries.STRUCTURE_FEATURES, MODID, existingFileHelper);
+        }
+
+        @Override
+        public String getName()
+        {
+            return "StructureFeature Tags";
```

Comment:
I feel like there should be a space here `StructureFeature` ->  `Structure Feature`